src/test/regress/expected/foreign_key_to_reference_table.out

```diff
@@ -342,7 +321,7 @@ SELECT run_command_on_workers($$CREATE TYPE fkey_reference_table.composite AS (k
 (2 rows)
 
 CREATE TABLE referenced_table(test_column composite, PRIMARY KEY(test_column));
-CREATE TABLE referencing_table(id int, referencing_composite composite);
+CREATE TABLE referencing_table(id int, referencing_composite composite, FOREIGN KEY (referencing_composite) REFERENCES referenced_table(test_column) ON DELETE CASCADE);
```

Comment:
I'm not a big fan of converting `ALTER TABLE` commands to fkey in `CREATE TABLE`. Our coverage for `ALTER TABLE` is obviously decreasing.